go.mod

```diff
@@ -14,8 +14,8 @@ require (
 	github.com/opentracing/opentracing-go v1.2.0 // indirect
 	github.com/philhofer/fwd v1.0.0 // indirect
 	github.com/sony/gobreaker v0.4.1
-	github.com/stretchr/testify v1.3.0
+	github.com/stretchr/testify v1.7.0
 	golang.org/x/time v0.0.0-20200630173020-3af7569d3a1e // indirect
 	golang.org/x/xerrors v0.0.0-20200804184101-5ec99f83aff1 // indirect
-	gopkg.in/DataDog/dd-trace-go.v1 v1.29.2
+	gopkg.in/DataDog/dd-trace-go.v1 v1.30.0
```

Comment:
Im new to go, what are these go.mod and go.sum files for? It seems like its listing dependencies? Im not sure.

Reply:
The `go.mod` file is basically the Go version of `package.json`.

`go.sum` contains checksums of all the project dependencies. It lets you check to see if any code has been modified since you last pulled it (for security).